app/src/main/java/com/example/chessclockk/usecase/TempoUseCase.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.chessclockk.usecase
+
+import android.content.SharedPreferences
+import javax.inject.Inject
+
+class TempoUseCase @Inject constructor(
+    private val sharedPreferences: SharedPreferences
+) {
+    fun saveTempo(gameTempo: String, bonusTempo: String) {
+        sharedPreferences
```

Comment:
Jeśli chodzi o clean architecture to w domain (use case) unikałbym zapisu. Ludzie dają to w data w repo. Ja robię sobie czasami jeszcze jedną klasę w data i nazywam "store". Dodatkowo to teraz często te use case nie są już potrzebne. Czyli TempoUseCase->TempoRepository (i ew. ->TempoStore)